Examples/Swift/ViewController.swift

```diff
@@ -270,12 +266,8 @@ extension ViewController: WaypointConfirmationViewControllerDelegate {
             navigationViewController.mapView!.userLocation!.coordinate,
             nextDestination,
         ])
-        options.includesSteps = true
-        options.routeShapeResolution = .full
-        options.profileIdentifier = navigationViewController.route.routeOptions.profileIdentifier
-        
-        // Adding the optional attribute `.congestionLevel` ensures the route line will show the congestion along the route line
-        options.attributeOptions = [.congestionLevel]
+            // Dont forget to apply `preferredOptions`. This adds the necessary options needed to make a navigation experience 💯.
+            .preferredOptions
```

Comment:
I agree about the overriding. However, [coordinates] won't bridge to Obj-C, how about a class function like `preferredOptions(waypoints:profileIdentifier:)`?